CTP/README.md

```diff
@@ -122,40 +122,62 @@ This ``Quick Start`` is only for user for reference about how to use ``CTP`` to
   
 - **SHELL**
   - Prepare
-	* Prepare at least two accounts (e.g., one account ``controller`` as controller, another account ``shell_instance1`` as test instance)
-	* Check out CTP for each account and configure environment variables for CTP ``controller`` and test instance ``shell_instance1``
-	
-	  ```
+        * A single instance local test

```

Comment:
A single instance local test   => For local test
The multiple instances test => For multiple instances test